.cqfd/docker/Dockerfile

```diff
@@ -12,6 +12,7 @@ ENV LC_ALL en_US.UTF-8
 RUN set -x \
     && apt-get update \
     && apt-get install -y \
+        ansible \
```

Comment:
I do not agree with this: we are not sure that Ansible will be packaged in 2.10 version in 22.04. Using Pip, we can specify the version we want. You should remove these lines.